__tests__/app.test.js

```diff
@@ -0,0 +1,38 @@
+const app = require("../app");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const testData = require("../db/data/test-data");
+
+const request = require("supertest");
+
+beforeEach(() => {
+  return seed(testData);
+});
+
+afterAll(() => {
+  return db.end();
+});
+
+describe("GET /api/not-a-route", () => {
+  test("404: returns URL not found message", () => {
+    return request(app)
+      .get("/api/not-a-route")
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("URL not found");
+      });
+  });
+});
+
+describe("GET /api/topics", () => {
+  test("200: sends an array of topics to the client", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
+        expect(Object.keys(body.topics[0])).toEqual(
+          expect.arrayContaining(["slug", "description"])
```

Comment:
This is only checking that the 1st topic object has the keys 'slug' and 'description'.

We want to test that we are getting back an array containing all of the available topic objects.